Update faqs.md (#293)
exercism/docs: using/faqs.md

We now have a [dedicated Teams site](https://teams.exercism.org).

## Why have my unlocked exercises in the old site become locked?

New in Exercism 3.0 is Learning Mode and this is the default for any user from the old site:

> * In Learning Mode there is a set of syllabus exercises to complete, which, as completed, unlock other exercises. If you are a returning user, who is used to previous versions of Exercism, you might find that other previously unlocked but unopened exercises have now become locked. To access those, without needing to go through the new syllabus easy exercises, you need to switch to Practice Mode.

Reviewer: It does feel slightly that we almost recommend people to switch to Practice Mode. Maybe we could phrase this slightly differently to encourage people to actually do the corresponding syllabus exercises first?

Reviewer 2: Yeah, we'd generally much prefer people to go through the Concept Exercises as they're designed to only take 5mins each for a fluent developer, so the whole lot can be done in an hour or so.

```suggestion
- In Learning Mode there is a set of Learning Exercises to complete. 
  Completing these exercises unlocks further Learning Exercises and also Practice Exercises. 
  If you are a returning user, who is used to previous versions of Exercism, you mat find that exercises that were previously available have now become locked. 
  While we recommend solving the Learning Exercises to unlock them (each should take 5mins to solve if you are fluent in a language), you can instead disable Learning Mode and switch to Practice Mode.
```

Author: Apart from the typos that change looks good to me. Note however different tracks have a different amount of concepts. I had no issue in F# but there were so many more in C# that this became frustrating.

Author: I can't see how to fix your typo, (since I only can edit my last patch fork), that is:

Line 55:  - Completing the unlock further Learning Exercises and also Practice Exercises. 

change to "- Completing a Syllabus Exercise unlocks further Learning Exercises and also Practice Exercises. "

Author: If you want lapsed users to re-start using Exercism, don't make it frustrating for them.

Author: I can't resolve this until the typo noted on line 55 is fixed

Reviewer 2: Updated.

> I can't see how to fix your typo

You can always accept a suggestion, then make another. Or you make your own suggestion, that's identical to mine with the fix, then accept it.

> * Note that in either Practice or Learning Mode, one can ask for mentoring on a particular exercise.

Reviewer 2: ```suggestion
- Note that in either Practice or Learning Mode, one can ask for mentoring on a particular exercise.
```

> * To switch modes, go to the Track Overview Page and click on the More "..." icon on the top right hand side and select "Disable Learning Mode". (Please note that it currently takes a while for the current mode to be reported in your Tracks page).

Reviewer 2: ```suggestion
- To switch modes, go to the Track Overview Page and click on the More "..." icon on the top right hand side and select "Disable Learning Mode".
```

To clarify: It doesn't take a while. The tracks page lists which tracks *have* learning mode, not the mode you're in. There's a discussion on this on the issue you posted on on exercism/exercism

Author: That is very misleading. I switched to practice mode and expected to see the change in Tracks but did not. This is a point of confusion.

Author: There is duplication in the tracks listing. If a track has zero concepts then it is "New". What does that mean especially for old i.e. long existing tracks? For tracks with Concepts >0  it has "learning mode" - what is the point this is confusing.

We now have a [dedicated Teams site](https://teams.exercism.org).

## Why have my unlocked exercises in the old site become locked?

New in Exercism 3.0 is Learning Mode and this is the default for any user from the old site:

- In Learning Mode there is a set of Learning Exercises to complete. 
  Completing these exercises unlocks further Learning Exercises and also Practice Exercises. 

>   If you are a returning user, who is used to previous versions of Exercism, you mat find that exercises that were previously available have now become locked. 

Reviewer: ```suggestion
  If you are a returning user, who is used to previous versions of Exercism, you may find that exercises that were previously available have now become locked. 
```

>   While we recommend solving the Learning Exercises to unlock them (each should take 5mins to solve if you are fluent in a language), you can instead disable Learning Mode and switch to Practice Mode.

Reviewer: ```suggestion
  While we recommend solving the Learning Exercises to unlock them (each should take around 5 minutes to solve if you are fluent in a language), you can instead disable Learning Mode and switch to Practice Mode.
```
